src/pages/HomePage.tsx

```diff
@@ -0,0 +1,16 @@
+import { WrappedDocument } from "@govtechsg/open-attestation";
+import React, { useState } from "react";
+import { DocumentRenderer } from "../components/DocumentRenderer";
+import { DropZone } from "../components/DropZone";
+
+export const HomePage: React.FunctionComponent = () => {
+  const [rawDocument, setRawDocument] = useState<WrappedDocument>();
+
+  return (
+    <div className="bg-gray-100">
+      <h1 className="mb-20 text-xl">Home page</h1>
+      <DropZone setRawDocument={(doc) => setRawDocument(doc)} />
```

Comment:
for the next mr, can u just center the dropzone or make it full width (and in that case center the content in the dropaone, if not already done) ?